### DISCUSSION TITLE
Broken routing on package page

### DISCUSSION THREAD

**Discussion by sergeysova** [npm/feedback#153] [Website] [answered]
Steps:
1. Open package page as the owner
1. Click on the "Versions" tab
2. Url doesn't change
3. Click on the "Settings" tab
4. Url changed
5. Then click on the "Versions" tab again
6. Url changed with `?activeTab=versions`
7. Now click on "Readme"
8. Url doesn't change
9. Reload the page, you are on the "Versions" tab
10. Click on "Readme"
11. Empty page
12. WTF is going on with routing?

[Video](https://t.me/sergeysova/282)

**Comment by MylesBorins** [comment, marked as answer]
Hey @sergeysova, seems like navigator is not getting updated as expected when clicking on specific tabs while following your steps... definitely not ideal or expected. I've created an internal bug to chat with the team to figure out if this behavior is intentional and what we can do to improve the status quo

**Reply by MylesBorins** [reply]
@sergeysova this is not something that we are going to be able to prioritize fixing in the short term. We have this bug tracked in a backlog but I cannot give a timeline on fixing, apologies.
